src/App.tsx

```diff
@@ -1,156 +1,98 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { TodoAppHeader } from './components/TodoAppHeader/TodoAppHeader';
+import { TodoAppContent } from './components/TodoAppContent/TodoAppContent';
+import { TodoAppFooter } from './components/TodoAppFooter/TodoAppFooter';
+import { Notifications } from './components/Notifications/Notifications';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterType';
+import { ErrorType } from './types/ErrorType';
+
+const USER_ID = 10308;
+
+const prepareTodos = (todoList: Todo[] | null, filterType: FilterType) => (
+  todoList?.filter(todo => {
+    switch (filterType) {
+      case FilterType.ACTIVE:
+        return !todo.completed;
+
+      case FilterType.COMPLETED:
+        return todo.completed;
+
+      default:
+        return true;
+    }
+  })
+);
+
+const getActiveTodosCount = (todoList: Todo[] | null) => (
+  todoList?.filter(todo => !todo.completed).length || 0
+);
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[] | null>(null);
+  const [filterType, setFilterType] = useState(FilterType.ALL);
+  const [errorType] = useState(ErrorType.NONE);
+
+  const getTodoList = useCallback(async () => {
+    const todos = await client.get<Todo[]>('/todos?userId=10308');
```

Comment:
make a separate function for it
```suggestion
    const todos = await getTodos();
```